sections/git_status_options.zsh

```diff
@@ -0,0 +1,22 @@
+#
+# Git status shared
+#
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GIT_STATUS_SHOW="${SPACESHIP_GIT_STATUS_SHOW=true}"
+SPACESHIP_GIT_STATUS_PREFIX="${SPACESHIP_GIT_STATUS_PREFIX=" ["}"
+SPACESHIP_GIT_STATUS_SUFFIX="${SPACESHIP_GIT_STATUS_SUFFIX="]"}"
+SPACESHIP_GIT_STATUS_COLOR="${SPACESHIP_GIT_STATUS_COLOR="red"}"
+SPACESHIP_GIT_STATUS_UNTRACKED="${SPACESHIP_GIT_STATUS_UNTRACKED="?"}"
+SPACESHIP_GIT_STATUS_ADDED="${SPACESHIP_GIT_STATUS_ADDED="+"}"
+SPACESHIP_GIT_STATUS_MODIFIED="${SPACESHIP_GIT_STATUS_MODIFIED="!"}"
+SPACESHIP_GIT_STATUS_RENAMED="${SPACESHIP_GIT_STATUS_RENAMED="»"}"
```

Comment:
I was actually thinking this would be useful for supporting things like #359 — you'd have something like

```
CUSTOM_GIT_SECTION_ADDED="${CUSTOM_GIT_SECTION_ADDED="$SPACESHIP_GIT_STATUS_ADDED"}"
```

`git_status_options.zsh` would be have the basics, and any section might use all or just some of them. That way your `.zshrc` could just have

```
SPACESHIP_GIT_STATUS_ADDED="added yeah!"
```

instead of

```
SPACESHIP_GIT_STATUS_ADDED="added yeah!"
CUSTOM_GIT_SECTION_ADDED="added yeah!"
```

Reply:
I see, but you wouldn't use two sections at the same time, would you? You'd choose one of the sections, then look up its available options and what they do, and configure to your liking 🙂

Reply:
Ha true. My thought was this would make it easier for people to try out other git prompts — they could simply change the line in the git prompt array, without changing a bunch of variable names. It also gives hypothetical git prompt developers a starting place

That said I don't strong feelings about it :) maybe we can get a third opinion

Reply:
Agreed, my final note would be that different modules could (and probably will) reuse the same variable names anyway, for example in my custom module I did use `SPACESHIP_GIT_STATUS_ADDED` variable name 🙂